### DISCUSSION TITLE
Neustart CCU3 über Button in Home assistant

### DISCUSSION THREAD

**Discussion by Daniel981a** [SukramJ/aiohomematic#765] [General]
Hallo zusammen,

wie kann ich über einen Button in Home Assistant den Neustart der CCU3 ermöglichen?
Vielleicht kann mir jemand helfen. Ich verzweifle...

Grüße

Daniel

**Comment by SukramJ** [comment]
Da die mir bekannten Homematic APIs diese Funktion nicht bieten, wird das von der Integration nicht unterstützt.

**Comment by Daniel981a** [comment]
Ok danke.
Wie wäre es, evtl. über einen virtuellen Schalter in der CCU3 ein Programm auszuführen, das den Neustart durchführt?
Gesetzt, das ginge, kann ich einen virtuellen Schalter in Home assistant einfügen?

**Reply by SukramJ** [reply]
Sowohl Programme als auch virtuelle Schalter werden in HA angezeigt, sind aber standardmäßig deaktiviert.
